src/promptflow/promptflow/_trace/_start_trace.py

```diff
@@ -79,12 +79,12 @@ def _start_pfs(pfs_port) -> None:
 
     command_args = ["start", "--port", str(pfs_port)]
     if is_port_in_use(pfs_port):
-        _logger.warning(f"Service port {pfs_port} is used.")
-        if is_pfs_service_healthy(pfs_port) is True:
-            _logger.info(f"Service is already running on port {pfs_port}.")
-            return
-        else:
+        is_healthy, content = is_pfs_service_healthy(pfs_port)
+        if not is_healthy:
             command_args += ["--force"]
+        else:
+            _logger.info(f"Service is already running on port {pfs_port}, {content}")
```

Comment:
Does the `is_pfs_service_healthy`'s result only used for this log? Can we move the log part inside the function to avoid it?

Reply:
updated